src/components/screens/login/networkSelector/customNode/readMode.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { TertiaryButton } from '@toolbox/buttons';
+import Icon from '@toolbox/icon';
+import styles from '../networkSelector.css';
+
+const ReadMode = ({
+  dropdownRef, setMode,
+  storedCustomNetwork, customNetworkRemoved, networkSelected,
+}) => {
+  const edit = (e) => {
+    e.preventDefault();
+    setMode('edit');
+  };
+
+  const remove = (e) => {
+    e.preventDefault();
+    customNetworkRemoved();
+    setMode('edit');
+  };
+
+  const connect = () => {
+    networkSelected({
+      name: 'customNode',
+      initialSupply: 1,
+      address: storedCustomNetwork,
+    });
+    dropdownRef.current.toggleDropdown(false);
+  };
+
+  return (
+    <div className={`${styles.customNode} ${styles.readMode}`}>
+      <span
+        className={styles.title}
+        onClick={connect}
+      >
+        {storedCustomNetwork}
+      </span>
+      <div className={styles.actions}>
+        <TertiaryButton
+          onClick={edit}
+          size="m"
+        >
+          <Icon name="edit" />
+        </TertiaryButton>
+        <TertiaryButton
+          onClick={remove}
+          size="m"
+        >
+          <Icon name="remove" />
```

Comment:
incorrect formatting?

Reply:
`>` in the new line? That's a pretty popular formatting. It's set in our liner config too.